File: BookShop.ClientApp/src/app/models/authorSearchParams.ts
import { AuthorModel } from 'src/app/models/authorModel';

export class AuthorSearchParams {
    name: string;
    page: number;
    pageSize: number;
    authorsList: AuthorModel[];
    constructor() {
        this.name = '';
        this.authorsList = [];
    }
}

File: BookShop.BusinessLogic/Models/Authors/AuthorSearchParams.cs
using System.Collections.Generic;

namespace BookShop.BusinessLogic.Models
{
    public class AuthorSearchParams
    {
        public string Name { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }
        public List<AuthorModel> AuthorsList { get; set; }
    }
}
